app/waf/src/repositories/common.ts

```diff
@@ -0,0 +1,28 @@
+import {Entity} from '@loopback/repository';
```

Comment:
Should this file's name be `common.respository.ts`?

Reply:
When name it common.respository.ts at the beginning, I got an exception when starting the app. I did not know its exact root cause, but I guess maybe some code was attempting to recognize the repository definition in this file and to create tables in db.

Reply:
'''
Server is running at http://127.0.0.1:3000
Try http://127.0.0.1:3000/adcaas/v1/ping
(node:11281) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'definition' of undefined
    at new DefaultCrudRepository (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/repository/dist/repositories/legacy-juggler-bridge.js:66:40)
    at new CommonRepository (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/dist/src/repositories/common.repository.js:6:1)
    at value_promise_1.transformValueOrPromise.args (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/context/dist/resolver.js:44:16)
    at Object.transformValueOrPromise (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/context/dist/value-promise.js:231:16)
    at Object.instantiateClass (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/context/dist/resolver.js:39:34)
    at Binding._getValue (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/context/dist/binding.js:356:55)
    at resolution_session_1.ResolutionSession.runWithBinding.s (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/context/dist/binding.js:189:90)
    at value_promise_1.tryWithFinally (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/context/dist/resolution-session.js:69:53)
    at Object.tryWithFinally (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/context/dist/value-promise.js:162:18)
    at Function.runWithBinding (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/context/dist/resolution-session.js:69:32)
    at Binding.getValue (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/context/dist/binding.js:189:65)
    at WafApplication.getValueOrPromise (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/context/dist/context.js:531:36)
    at WafApplication.get (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/context/dist/context.js:471:27)
    at Promise.all.repoBindings.map.b (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/repository/dist/mixins/repository.mixin.js:179:58)
    at Array.map (<anonymous>)
    at WafApplication.migrateSchema (/Users/qzhao/github/zhaoqin-github/f5-adcaas-openstack/app/waf/node_modules/@loopback/repository/dist/mixins/repository.mixin.js:179:44)
(node:11281) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
'''

Reply:
Indeed. Seems lb4 check the folder to do the same action for all *.repository.ts.